hazelcast/src/main/java/com/hazelcast/internal/serialization/impl/SerializationServiceHelper.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.internal.serialization.impl;
+
+import com.hazelcast.core.PartitioningStrategy;
+import com.hazelcast.instance.OutOfMemoryErrorDispatcher;
+import com.hazelcast.internal.serialization.SerializationService;
+import com.hazelcast.nio.serialization.ByteArraySerializer;
+import com.hazelcast.nio.serialization.Data;
+import com.hazelcast.nio.serialization.HazelcastSerializationException;
+import com.hazelcast.nio.serialization.Serializer;
+import com.hazelcast.nio.serialization.StreamSerializer;
+
+import java.util.Collections;
+import java.util.Set;
+
+import static java.lang.Boolean.parseBoolean;
+
+public final class SerializationServiceHelper {
```

Comment:
Personally I'm always a bit careful introducing helpers/utils. Should this not be part of the SerializationService?